omero/conf.py

```diff
@@ -97,39 +97,48 @@
+    'common_source' : (omero_subs_github_root.format('omero-common', 'blob', 'v'+conf_autogen.version_common), ''),
+    'common_sourcedir' : (omero_subs_github_root.format('omero-common', 'tree', 'v'+conf_autogen.version_common), ''),
+    'romio_source' : (omero_subs_github_root.format('omero-romio', 'blob', 'v'+conf_autogen.version_romio), ''),
+    'romio_sourcedir' : (omero_subs_github_root.format('omero-romio', 'tree', 'v'+conf_autogen.version_romio), ''),
+    'renderer_source' : (omero_subs_github_root.format('omero-renderer', 'blob', 'v'+conf_autogen.version_renderer), ''),
+    'renderer_sourcedir' : (omero_subs_github_root.format('omero-renderer', 'tree', 'v'+conf_autogen.version_renderer), ''),
+    'server_source' : (omero_subs_github_root.format('omero-server', 'blob', 'v'+conf_autogen.version_server), ''),
+    'server_sourcedir' : (omero_subs_github_root.format('omero-server', 'tree', 'v'+conf_autogen.version_server), ''),
+    'blitz_source' : (omero_subs_github_root.format('omero-blitz', 'blob', 'v'+conf_autogen.version_blitz), ''),
+    'blitz_sourcedir' : (omero_subs_github_root.format('omero-blitz', 'tree', 'v'+conf_autogen.version_blitz), ''),
+    'java_gateway_source' : (omero_subs_github_root.format('omero-java-gateway', 'blob', 'v'+conf_autogen.version_gateway), ''),
+    'java_gateway_sourcedir' : (omero_subs_github_root.format('omero-java-gateway', 'tree', 'v'+conf_autogen.version_gateway), ''),
+    'matlab_source' : (omero_subs_github_root.format('omero-matlab', 'blob', 'v'+conf_autogen.version_matlab), ''),
+    'matlab_sourcedir' : (omero_subs_github_root.format('omero-matlab', 'tree', 'v'+conf_autogen.version_matlab), ''),
+    'insight_source' : (omero_subs_github_root.format('omero-insight', 'blob', 'v'+conf_autogen.version_matlab), ''),
+    'insight_sourcedir' : (omero_subs_github_root.format('omero-insight', 'tree', 'v'+conf_autogen.version_matlab), ''),
     # API links
     'javadoc' : (downloads_root + '/latest/omero5.4/api/%s', ''),
+    'javadoc_gateway_java' : (docs_root + '/omero-gateway/' + conf_autogen.version_gateway + '/javadoc/%s', ''),
+    'javadoc_blitz' : (docs_root + '/omero-blitz/' + conf_autogen.version_blitz + '/javadoc/%s', ''),
+    'javadoc_server' : (docs_root + '/omero-server/' + conf_autogen.version_server + '/javadoc/%s', ''),
+    'javadoc_renderer' : (docs_root + '/omero-renderer/' + conf_autogen.version_renderer + '/javadoc/%s', ''),
+    'javadoc_romio' : (docs_root + '/omero-romio/' + conf_autogen.version_romio + '/javadoc/%s', ''),
+    'javadoc_common' : (docs_root + '/omero-common/' + conf_autogen.version_common + '/javadoc/%s', ''),
+    'javadoc_model' : (docs_root + '/omero-model/' + conf_autogen.version_model + '/javadoc/%s', ''),
+    'slicedoc_blitz' : (docs_root + '/omero-blitz/' + conf_autogen.version_blitz + '/slice2html/%s', ''),
+    'slicedoc_server' : (docs_root + '/omero-server/' + conf_autogen.version_server + '/slice2html/%s', ''),
```

Comment:
Is there any?

Reply:
no I will remove it. This was prep before I went into the deep end of the doc